pkg/project/v2/project_loader.go

```diff
@@ -109,9 +112,13 @@ func LoadProjects(ctx context.Context, fs afero.Fs, loaderContext ProjectLoaderC
 		project, errs := loadProject(ctx, workingDirFs, loaderContext, projectDefinition, environments)
 
 		if len(errs) > 0 {
+			for _, err := range errs {
+				reporter.ReportLoading(report.StateError, err, "", nil)
```

Comment:
Is it OK to leave `message` empty?

Reply:
I thought we either need error or message but not both at the same time. Do you think we should add a default message?

Reply:
Maybe we can set msg to err in the consumer if it's not set?

Reply:
Then we would always have two properties (message and error) with the same content for error events